atomicfu-common/src/main/kotlin/kotlinx/atomicfu/AtomicFU.common.kt

```diff
@@ -120,6 +131,79 @@ public inline fun <T> AtomicRef<T>.updateAndGet(function: (T) -> T): T {
     }
 }
 
+
+// ==================================== AtomicBoolean ====================================
+
+/**
+ * Atomic reference to a [Boolean] variable with volatile reads/writes via
+ * [value] property and various atomic read-modify-write operations
+ * like [compareAndSet] and others.
+ */
+public expect class AtomicBoolean {
+    /**
+     * Reading/writing this property maps to read/write of volatile variable.
+     */
+    public var value: Int
```

Comment:
It should be `Boolean`

This code should compile:
```
val flag = atomic(false)
flag.value = true
```

Reply:
Thanks 
Fixed.